core/dbt/parser/manifest.py

```diff
@@ -1141,6 +1141,24 @@ def process_metrics(self, config: RuntimeConfig):
 
     def process_saved_queries(self, config: RuntimeConfig):
         """Processes SavedQuery nodes to populate their `depends_on`."""
+        # Note: This will also capture various nodes which have been re-parsed
+        # because they refer to some other changed node, so there will be
+        # false positives. Ideally we would compare actual changes.
+        semantic_manifest_changed = False
+        SemanticManifestNode = Union[SavedQuery, SemanticModel, Metric]
```

Comment:
Nit: Can we put this definition in a constants file somewhere?

Reply:
Moved it to core/dbt/contracts/graph/nodes.py